NebulaModel/Packets/Planet/PlanetDataResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace NebulaModel.Packets.Planet
+{
+    public class PlanetDataResponse
+    {
+        public int[] PlanetDataIDs { get; set; }
+        public byte[] PlanetDataBytes { get; set; }
+        public int[] PlanetDataBytesLengths { get; set; }
+
+        public PlanetDataResponse() { }
+
+        public PlanetDataResponse(Dictionary<int, byte[]> planetData) 
+        {
+            this.PlanetDataIDs = planetData.Keys.ToArray();
+
+            // Can't use a jagged array because LNL serializer says no, so flattening and separate offset array it is
+            // TODO: Possibly register a type with the serializer instead of doing this manually
```

Comment:
We could go with a custom type or I could look into the LiteNetLib serialization code and add support for jagged arrays if it's something that we will need a lot.

Reply:
Let's leave it as is for now, and then if it comes up again we can add a type or update the serializer, it's not critical